app/controllers/admin/base_items_controller.rb

```diff
@@ -40,7 +42,9 @@ def show
 
   def destroy
     @base_item = BaseItem.includes(:items).find(params[:id])
-    if @base_item.items.any? && @base_item.destroy
+    if (@base_item.id = KitCreateService.find_or_create_kit_base_item!.id)
```

Comment:
Assignment inside a condition is pretty prone to bugs. Can we separate them out into two lines?

Reply:
I think this was a typo, fixed to `==` in commit [94030f7](https://github.com/rubyforgood/human-essentials/pull/4665/commits/94030f75aefe8cfa6ecc4e6155da4c8e18bb630f)